### PR TITLE
[mdf] Scrape Moksha + slightly more flexible default pron selector.

### PR DESCRIPTION
**Pull request by agutkin**
Strangely enough some (but not all) Moksha pages are not standard. The pronunciations don't reside under the regular list item elements ("li"), but under the paragraph elements ("p"). For an example, see the page for [ала](https://en.wiktionary.org/wiki/%D0%B0%D0%BB%D0%B0#Moksha)

Please let me know whether you'd prefer a custom extractor for this rather than changing the default template.

- [x] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

### CODE REVIEW

**Comment by kylebgorman** [comment]
Whether this should be a custom extractor or not depends on whether this breaks anything. Are you willing to run `data/src/./scrape.py` for a while and check for any regressions?

**Comment by agutkin** [comment]
Good idea. I'll try that.

**Comment by agutkin** [comment]
I've been running a full scrape for a while now. I presume I can just leave it running and report back the results when done.

**Comment by kylebgorman** [comment]
Yes indeed. No need to commit it, just check file sizes before and after I
suppose. (The README in data/ explains how to generate the tables.)

File: wikipron/config.py [hunk 1]
```diff
@@ -15,8 +15,9 @@
 
 # GH-49: Estonian and Slovak use @title = "wikipedia:{language} phonology".
 # GH-50: Korean has an extra "span" layer (for fonts) in //li[span[sup[a.
+# Some (but not all) Moksha pronunciations reside in //p[sup[a.
 _PRON_XPATH_SELECTOR_TEMPLATE = """
-//li[
+(//li|//p)[
```
Comment (jacksonllee) [inline wikipron/config.py:20]: As far as I can tell, this shouldn't break anything. (I imagine this is why Kyle tagged me for code review -- let me know if there's anything else.)

If we confirm this is the case, plus an entry in `CHANGELOG.md`, then this PR is good to go.

**Comment by kylebgorman** [comment]
Yes, what Jackson said, just curious whether it was obvious to him that anything would break from this. If Sasha says it's okay, we'll roll it out.

**Comment by kylebgorman** [comment]
@agutkin update please? Scrape go okay so far?

**Comment by agutkin** [comment]
It was almost ok. Will give an update in a bit.

**Comment by agutkin** [comment]
I ran it all the way up to the Czech at which point the server was rebooted by the administrators. 

The diff is provided below. By eyeballing it, it mostly looks ok modulo Armenian. For some reason it is not scraped well with a lot of entries removed. Please also note the 15k new Cantonese entries in `yue_phonemic.tsv`:

```
diff --git a/data/languages_summary.tsv b/data/languages_summary.tsv
index c93fc87..c9d82fd 100644
--- a/data/languages_summary.tsv
+++ b/data/languages_summary.tsv
@@ -1,55 +1,55 @@
 ady_phonetic.tsv       ady     Adygei; Adyghe  Adyghe  True    Phonetic        5123
 ady_phonetic_filtered.tsv      ady     Adygei; Adyghe  Adyghe  True    Phonetic_filtered       4895
-aar_phonemic.tsv       aar     Afar    Afar    True    Phonemic        339
-afr_phonemic.tsv       afr     Afrikaans       Afrikaans       True    Phonemic        1333
-afr_phonetic.tsv       afr     Afrikaans       Afrikaans       True    Phonetic        120
-alb_phonemic.tsv       alb     Albanian        Albanian        True    Phonemic        1329
-alb_phonetic.tsv       alb     Albanian        Albanian        True    Phonetic        799
-gsw_phonemic.tsv       gsw     Swiss German    Alemannic German        True    Phonemic        305
+aar_phonemic.tsv       aar     Afar    Afar    True    Phonemic        715
+afr_phonemic.tsv       afr     Afrikaans       Afrikaans       True    Phonemic        1683
+afr_phonetic.tsv       afr     Afrikaans       Afrikaans       True    Phonetic        121
+alb_phonemic.tsv       alb     Albanian        Albanian        True    Phonemic        1448
+alb_phonetic.tsv       alb     Albanian        Albanian        True    Phonetic        814
+gsw_phonemic.tsv       gsw     Swiss German    Alemannic German        True    Phonemic        307
 ale_phonemic.tsv       ale     Aleut   Aleut   True    Phonemic        104
-grc_phonemic.tsv       grc     Ancient Greek (to 1453) Ancient Greek   True    Phonemic        84667
-ara_phonemic.tsv       ara     Arabic  Arabic  False   Phonemic        6379
-ara_phonetic.tsv       ara     Arabic  Arabic  False   Phonetic        112
-arc_hebr_phonemic.tsv  arc     Imperial Aramaic (700-300 BCE); Official Aramaic (700-300 BCE)  Aramaic (Hebrew)        False   Phonemic        1153
-arm_e_phonetic.tsv     arm     Armenian        Armenian (Eastern Armenian)     True    Phonetic        13702
-arm_e_phonetic_filtered.tsv    arm     Armenian        Armenian (Eastern Armenian)     True    Phonetic_filtered       13258
-arm_w_phonetic.tsv     arm     Armenian        Armenian (Western Armenian)     True    Phonetic        742
-arm_w_phonetic_filtered.tsv    arm     Armenian        Armenian (Western Armenian)     True    Phonetic_filtered       599
-rup_phonetic.tsv       rup     Macedo-Romanian Aromanian       True    Phonetic        143
-asm_phonemic.tsv       asm     Assamese        Assamese        False   Phonemic        2214
-ast_phonetic.tsv       ast     Asturian        Asturian        True    Phonetic        131
-aze_latn_phonemic.tsv  aze     Azerbaijani     Azerbaijani (Latin)     True    Phonemic        225
-aze_latn_phonetic.tsv  aze     Azerbaijani     Azerbaijani (Latin)     True    Phonetic        2579
-bdq_phonemic.tsv       bdq     Bahnar  Bahnar  True    Phonemic        149
-ban_bali_phonemic.tsv  ban     Balinese        Balinese (Balinese)     True    Phonemic        178
+grc_phonemic.tsv       grc     Ancient Greek (to 1453) Ancient Greek   True    Phonemic        89549
+ara_phonemic.tsv       ara     Arabic  Arabic  False   Phonemic        7184
+arc_hebr_phonemic.tsv  arc     Imperial Aramaic (700-300 BCE); Official Aramaic (700-300 BCE)  Aramaic (Hebrew)        False   Phonemic        1156
+arm_e_phonetic.tsv     arm     Armenian        Armenian (Eastern Armenian)     True    Phonetic        353
+arm_e_phonetic_filtered.tsv    arm     Armenian        Armenian (Eastern Armenian)     True    Phonetic_filtered       274
+arm_w_phonetic.tsv     arm     Armenian        Armenian (Western Armenian)     True    Phonetic        353
+arm_w_phonetic_filtered.tsv    arm     Armenian        Armenian (Western Armenian)     True    Phonetic_filtered       274
+rup_phonetic.tsv       rup     Macedo-Romanian Aromanian       True    Phonetic        147
+asm_phonemic.tsv       asm     Assamese        Assamese        False   Phonemic        2350
+ast_phonetic.tsv       ast     Asturian        Asturian        True    Phonetic        133
+aze_latn_phonemic.tsv  aze     Azerbaijani     Azerbaijani (Latin)     True    Phonemic        244
+aze_latn_phonetic.tsv  aze     Azerbaijani     Azerbaijani (Latin)     True    Phonetic        2757
+bdq_phonemic.tsv       bdq     Bahnar  Bahnar  True    Phonemic        161
+ban_bali_phonemic.tsv  ban     Balinese        Balinese (Balinese)     True    Phonemic        186
 bak_phonemic.tsv       bak     Bashkir Bashkir True    Phonemic        108
-bak_phonetic.tsv       bak     Bashkir Bashkir True    Phonetic        2006
-baq_phonemic.tsv       baq     Basque  Basque  True    Phonemic        1256
-baq_phonetic.tsv       baq     Basque  Basque  True    Phonetic        189
-bel_phonetic.tsv       bel     Belarusian      Belarusian      True    Phonetic        3073
-ben_phonemic.tsv       ben     Bengali Bengali False   Phonemic        770
-ben_phonetic.tsv       ben     Bengali Bengali False   Phonetic        155
+bak_phonetic.tsv       bak     Bashkir Bashkir True    Phonetic        2049
+baq_phonemic.tsv       baq     Basque  Basque  True    Phonemic        1690
+baq_phonetic.tsv       baq     Basque  Basque  True    Phonetic        188
+bel_phonetic.tsv       bel     Belarusian      Belarusian      True    Phonetic        3197
+ben_phonemic.tsv       ben     Bengali Bengali False   Phonemic        791
+ben_phonetic.tsv       ben     Bengali Bengali False   Phonetic        169
+bcl_phonemic.tsv       bcl     Central Bikol   Bikol Central   True    Phonemic        100
 pcc_phonemic.tsv       pcc     Bouyei  Bouyei  True    Phonemic        105
-bre_phonemic.tsv       bre     Breton  Breton  True    Phonemic        485
-kxd_phonemic.tsv       kxd     Brunei  Brunei Malay    True    Phonemic        342
-bul_phonemic.tsv       bul     Bulgarian       Bulgarian       True    Phonemic        32486
-bul_phonemic_filtered.tsv      bul     Bulgarian       Bulgarian       True    Phonemic_filtered       32401
-bul_phonetic.tsv       bul     Bulgarian       Bulgarian       True    Phonetic        5274
-bur_phonemic.tsv       bur     Burmese Burmese False   Phonemic        4325
+bre_phonemic.tsv       bre     Breton  Breton  True    Phonemic        494
+kxd_phonemic.tsv       kxd     Brunei  Brunei Malay    True    Phonemic        346
+bul_phonemic.tsv       bul     Bulgarian       Bulgarian       True    Phonemic        32120
+bul_phonemic_filtered.tsv      bul     Bulgarian       Bulgarian       True    Phonemic_filtered       32013
+bul_phonetic.tsv       bul     Bulgarian       Bulgarian       True    Phonetic        6131
+bur_phonemic.tsv       bur     Burmese Burmese False   Phonemic        4471
+yue_phonemic.tsv       yue     Yue Chinese     Cantonese       False   Phonemic        15023
 crx_phonemic.tsv       crx     Carrier Carrier False   Phonemic        175
-cat_phonemic.tsv       cat     Catalan; Valencian      Catalan True    Phonemic        51341
-ceb_phonemic.tsv       ceb     Cebuano Cebuano True    Phonemic        292
-nya_phonemic.tsv       nya     Nyanja  Chichewa        True    Phonemic        821
-cmn_hani_phonemic.tsv  cmn     Mandarin Chinese        Chinese (Han)   False   Phonemic        125901
+cat_phonemic.tsv       cat     Catalan; Valencian      Catalan True    Phonemic        55617
+ceb_phonemic.tsv       ceb     Cebuano Cebuano True    Phonemic        319
+nya_phonemic.tsv       nya     Nyanja  Chichewa        True    Phonemic        823
+cmn_hani_phonemic.tsv  cmn     Mandarin Chinese        Chinese (Han)   False   Phonemic        131589
 cho_phonemic.tsv       cho     Choctaw Choctaw True    Phonemic        112
-nci_phonemic.tsv       nci     Classical Nahuatl       Classical Nahuatl       True    Phonemic        836
-nci_phonetic.tsv       nci     Classical Nahuatl       Classical Nahuatl       True    Phonetic        1190
-syc_phonetic.tsv       syc     Classical Syriac        Classical Syriac        False   Phonetic        5934
-cop_phonemic.tsv       cop     Coptic  Coptic  True    Phonemic        256
-cor_phonemic.tsv       cor     Cornish Cornish True    Phonemic        108
-cor_phonetic.tsv       cor     Cornish Cornish True    Phonetic        569
-cos_phonemic.tsv       cos     Corsican        Corsican        True    Phonemic        134
+nci_phonemic.tsv       nci     Classical Nahuatl       Classical Nahuatl       True    Phonemic        822
+nci_phonetic.tsv       nci     Classical Nahuatl       Classical Nahuatl       True    Phonetic        1388
+syc_phonetic.tsv       syc     Classical Syriac        Classical Syriac        False   Phonetic        5947
+cop_phonemic.tsv       cop     Coptic  Coptic  True    Phonemic        383
+cor_phonemic.tsv       cor     Cornish Cornish True    Phonemic        124
+cor_phonetic.tsv       cor     Cornish Cornish True    Phonetic        681
+cos_phonemic.tsv       cos     Corsican        Corsican        True    Phonemic        187
 cze_phonetic.tsv       cze     Czech   Czech   True    Phonetic        25740
 dlm_phonemic.tsv       dlm     Dalmatian       Dalmatian       True    Phonemic        176
 dan_phonemic.tsv       dan     Danish  Danish  True    Phonemic        3450
```

**Comment by kylebgorman** [comment]
Those are both known issues. (I have since solved Armenian and am working
on Cantonese.)

Okay, I think this is go. Any objections?

**Comment by agutkin** [comment]
I haz no object. One issue though: I've been wondering whether making these extraction templates more sophisticated may adversely affect the performance. WDYT? No idea how this over-the-network-HTML-thing functions.

**Comment by kylebgorman** [comment]
No idea myself. I don't think this one disjunction is going to be the big
issue. The network IO is the limitation I think.

**Comment by kylebgorman** [comment]
Add note to changelog and I'll approve, Sasha.

**Comment by agutkin** [comment]
Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ Unreleased
 -   Added Latgalian (`ltg`). (\#296)
 -   Added draft of covering grammar script. (\#297)
 -   Added Portuguese `.phones` files and re-scraped data. (\#290, \#304)
+-   Added Moksha (`mdf`) and modified the pron selector. (\#295)
```
Comment (kylebgorman) [inline CHANGELOG.md:62]: Sorry to nitpick but I think the language should go in this section and the change to the pron selector should go in "Under `wikipron/` side of the changelog.

**Comment by agutkin** [comment]
Done again.

**Review by kylebgorman — APPROVED** [review]
LGTM

**Pull request merged**